src/drivers/mram/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_library(mram mram.c)
```

Comment:
to compile this code, remember to add the subdirectory into the parent directories' CMakeList.txt